spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -44,7 +58,8 @@
 
 RSpec.shared_examples 'enrollment encountering an exception' do |exception_class: nil,
                                                                 exception_message: nil,
-                                                                reason: 'Request exception'|
+                                                                reason: 'Request exception',
+                                                                response_json: {}|
```

Comment:
Doesn't look like `response_json` is used in these shared examples. Should we revert this change?